buildSrc/src/main/java/net/neoforged/neodev/NeoDevPlugin.java

```diff
@@ -26,6 +27,7 @@
 import net.neoforged.nfrtgradle.DownloadAssets;
 import net.neoforged.nfrtgradle.NeoFormRuntimePlugin;
 import net.neoforged.nfrtgradle.NeoFormRuntimeTask;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Let's not use commons lang. If really needed, copy this instead: https://github.com/neoforged/ModDevGradle/blob/df0bb357418149add41a39ac1d5f73a1d5adcf9a/src/main/java/net/neoforged/moddevgradle/internal/utils/StringUtils.java#L32.

Reply:
I wasn't even sure where the dependency on commons-lang comes from... So agreed, will do something else.